File: Examples/Java/SDK/src/main/java/com/aspose/imaging/cloud/examples/images/UpdateJpgProperties.java
package com.aspose.imaging.cloud.examples.images;

import com.aspose.imaging.cloud.examples.Utils;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class UpdateJpgProperties {

    public static void main(String... args) throws IOException {
        String input = "Sample1.jpg";
        String output = "Sample2.jpg";
        Path inputFile = Utils.getPath(UpdateJpgProperties.class, input);
        Path outputFile = Utils.getPath(UpdateJpgProperties.class, output);

        Integer quality = 100;
        String compressionType = "progressive";
        Boolean fromScratch = false;
        String outPath = "";

        Utils.getStorageSdk().PutCreate(
                input,
                null,
                Utils.STORAGE,
                inputFile.toFile()
        );

        Utils.getImagingSdk().GetImageJpg(
                input, 
                quality, 
                compressionType, 
                fromScratch, 
                outPath, 
                Utils.FOLDER,
                Utils.STORAGE
        );

        com.aspose.storage.model.ResponseMessage sr
                = Utils.getStorageSdk().GetDownload(
                        input,
                        null,
                        Utils.STORAGE
                );

        Files.copy(sr.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);

    }
}

File: Examples/Node.js/REST/images/UpdateJpgProperties.js
var utils = require("../utils.js");
var fs = require('fs');

var input_file = 'sample1.jpg';
var inputPath = getPath(__filename, input_file);
var output_file = 'output.jpg';
var outputPath = getPath(__filename, output_file);

var input_buffer = fs.readFileSync(inputPath);

var method = 'PUT';
var request_url = 'storage/file/' + 'sample1.jpg';
UploadFileBinary(
    method,
    Sign(request_url),
    input_buffer,
    function (response) {
        if (response.Status == 'OK') {
            console.log(inputPath, 'has been uploaded');
        }
        var method = 'GET';
        var request_url ='imaging/sample1.jpg/jpg?quality=100&compressionType=progressive&outPath=' + output_file;;

        ProcessCommandContent(
            method,
            Sign(request_url),
            null,
            function(buffer) {
                fs.writeFileSync(outputPath, buffer);
            }
        );
    }
);